packages/admin/admin/src/theme/componentsTheme/MuiDataGrid.tsx

```diff
@@ -129,6 +129,9 @@ export const getMuiDataGrid: GetMuiComponentTheme<"MuiDataGrid"> = (component, {
             "&:focus-within": {
                 outline: "none",
             },
+            "&--last .MuiDataGrid-columnSeparator": {
```

Comment:
I am not sure if we **always** want to remove/hide the last column separator. 

Furthermore, if we remove the last column Separator, it is not possible anymore to resize the last column.

Should we probably make this configurable, maybe we could bring this to the column definition?